BMI added

Kushal-2002 opened the pull request:
## Description 📜

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.
Fixes #22 
I have added a BMI calculator using html, css and javascript.

<hr>
 
## Checklist ✅

<!----Please delete options that are not relevant.And in order to tick the check box just but x inside them for example [x] like this----->

- [✅ ] I follow [Contributing Guidelines](https://github.com/vasu-1/CalcHub/blob/main/.github/ContributingGuidelines.md) & [Code of conduct](https://github.com/vasu-1/CalcHub/blob/main/.github/CODE_OF_CONDUCT.MD) of this project.
- [✅ ] I have performed a self-review of my own code or work.
- [✅ ] I have commented my code, particularly in hard-to-understand areas.
- [✅ ] My changes generates no new warnings.
- [✅ ] I'm GSSOC'22 contributor
- [ ✅] Have you added a working video of your calculator ?

<hr>

<!----Please delete options that are not relevant.And in order to tick the check box just but x inside them for example [x] like this----->

## Screenshots / Gif (Optional) 📸

![BMI Calculator - Google Chrome 3_1_2022 1_19_40 PM](https://user-images.githubusercontent.com/90457566/156127268-1e0824e2-6a6b-4b43-a0b6-65157399ef9f.png)

<hr>

vasu-1 commented:
1. mention the issue number
2. rename the file `readme.md.txt` to `readme.md`
3. add necessary details in the readme
4. add your calculator link in the main readme
5. Also add screenshot to readme

Kushal-2002 commented:
@vasu-1 I have made the necessary changes in the readme file. Can you please tell me  how can I add calculator link in main readme?

vasu-1 commented:
> @vasu-1 I have made the necessary changes in the readme file. Can you please tell me how can I add calculator link in main readme?

That i will do from my end

vasu-1 commented:
delete `readme.md.txt`
And use some markdown in your `readme`
Also don't put(upload) binary(image) file in repo instead you can directly copy the image and paste in readme.md file

Kushal-2002 commented:
Please can you review and merge my pull request for issue #22.

vasu-1 commented:
> delete `readme.md.txt` And use some markdown in your `readme` Also don't put(upload) binary(image) file in repo instead you can directly copy the image and paste in readme.md file
Do this changes

Kushal-2002 commented:
Thank you for the instructions. I have made all the changes and updated my files.

vasu-1 reviewed (changes requested):
Binary files need to be deleted !!!
There is still one image file

vasu-1 commented:
Also add your file link in main readme.md
Make sure to update your cloned repo

Kushal-2002 commented:
@vasu-1 I have deleted the binary image file and also added my calculator link in the main Readme file.

mansi-garg-12 reviewed (changes requested):
Please remove the .png file and add it on README.md or past a link of google drive.

vasu-1 commented:
![image](https://user-images.githubusercontent.com/76911582/156615337-c4df5f1c-dc2e-4a22-ae45-a41587be9fb2.png)

Image file is still there,
Please remove it
And upload it directly by simply copy and paste in readme

Kushal-2002 commented:
I have removed the .png file and added a google drive image link in the main README.md

vasu-1 commented on Calculators/BMI_Calculator/readme.md:
```diff
@@ -0,0 +1,3 @@
+# BMI Calculator
+## I have added the BMI calculator.
```
you need to add the image here !
And you can just directly copy your image and paste it in readme file itself.. just like text copy and paste

Kushal-2002 replied:
I have tried to copy and paste my image several times but it is not happening. So I have added the google drive link.

Kushal-2002 commented:
I have added the image to my readme.md file and deleted it from previous place. But for the image I am unable to do direct copy and paste so I have to added the google drive link.

mansi-garg-12 reviewed (approved):
checked

vasu-1 commented:
Sorry for that !
But there are lot's of conflicts !
I accidently deleted the readme.md

What you can do is, you can just close this pr, delete this cloned repo and again clone it, and than make the folder and do the changes,
After that, make another pr for the same !!!!!!!!!!!!!!

if you have any queries reach out to me at discord !!

vasu-1 commented:
Please make another pr !

Kushal-2002 commented:
@vasu-1 do I need to make a new issue and make PR over there? Also please tell me how can I reach out to you via discord?
